modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPFailureTest.java

```diff
@@ -120,6 +139,19 @@ public void addIdPWithDuplicateProperties() throws IOException {
         validateErrorResponse(response, HttpStatus.SC_BAD_REQUEST, "IDP-60025");
     }
 
+    @Test
+    public void testAddIdPWithDuplicatedPropertyKeys() throws IOException {
```

Comment:
Lets exsiting idp creation json and build an authenticator for scenrios we are testing. Adding json for each test is not scable and also if payload format change that we have to update each.

Reply:
Applied to other test classes.

Reply:
This is fixed with 0af989440365ad1dc7105405492c477bd669f9c3